docs/apis/4-balance-v2-sse.md

```diff
@@ -192,83 +182,25 @@ the `displayProps` before inferring data from other fields present in a TokenBre
 
 ## Events and Payloads
 
-### start
-
-Signals that the server is ready and will start sending data down the wire.
-The data payload can be safely ignored.
-
 ### end
 
 Signals that the server has done all the needed work and will no longer send any events.
 The data payload can be safely ignored.
 
 **Important**: Please close the connection on reception of the 'end' event.
 
-### totals
-
-Signifies a calculated balance total of a given address.
-
-This event expects the client to SQUASH local state as it computes all that is needed and
-should be updated as a whole.
-
-Totals will always contain the final calculation of all possible balances as well
-as some minor statistics.
-
-The sent payload is `TotalsPayload`.
-
-```typescript
-type TotalsPayload = {
-  categoriesTotal: {
-    claimable: number;
-    debt: number;
-    deposits: number;
-    vesting: number;
-    nft: number;
-    wallet: number;
-    locked: number;
-  };
-  netTotal: number;
-  netTotalWithNfts: number;
-  assetTotal: number;
-  debtTotal: number;
-  networkTotals: {
-    // NOTE: the key value pair will ONLY appear if a balance is found
-    [network_name: string]: number;
-  };
-  claimablePerNetwork: {
-    // NOTE: the key value pair will ONLY appear if a balance is found
-    [network_name: string]: number;
-  };
-  stats: {
-    topHoldings: Array<{
-      appId: string;
-      label: string;
-      balanceUSD: number;
-      key: string;
-      pctHolding: number;
-    }>;
-    topHoldingsWithNfts: Array<{
-      appId: string;
-      label: string;
-      balanceUSD: number;
-      key: string;
-      pctHolding: number;
-    }>
-  };
-};
-```
-
-### category
+### balance
 
-Represents a new/updated category.
+Represents a new/updated balance result.
+This balance could be either for a wallet token, an app participation an NFT.
```

Comment:
> This balance is for a wallet token, an app position, or an NFT.

Maybe it's a bit more clear? We should probably use `position` instead of participation.